js/src/setup-ads/ads-stepper/setup-accounts/google-ads-account-card/connect-ads/index.scss

```diff
@@ -2,4 +2,12 @@
 	.app-select-control {
 		flex-grow: 1;
 	}
+
+	.wcdl-subsection-body {
+		margin-bottom: 8px;
+
+		a {
+			text-decoration: none;
```

Comment:
:question: Do we really need to remove this underline? In all the other flows we do have underlined, and I think it'd be nice to have a conventional and consistent experience.

Reply:
Ahh you're right, for example the Learn more link in the Reclaim URL Card below:

![image](https://user-images.githubusercontent.com/417342/160143323-48ac00f9-cf11-43fa-bb28-c932cfcdc35e.png)

I'll make the changes to make it consistent then. 🙂

Reply:
Done in [`64f64ae` (#1359)](https://github.com/woocommerce/google-listings-and-ads/pull/1359/commits/64f64ae0570616fe539dfc2915c2dd6dfbae2a6c).

![image](https://user-images.githubusercontent.com/417342/160144128-274b2b8e-71f2-427d-b1bd-6eddd3c1f014.png)